src/org.xtuml.bp.io.core/arc/gen_import_java.inc

```diff
@@ -505,7 +505,7 @@ public class ${class_name} extends CoreImport
             IPath projLoc = null;
                 projLoc = file.getProject().getLocation();
                 if (projLoc.isPrefixOf(curPath)) {
-                    projRelPath = curPath.removeFirstSegments(projLoc.segmentCount() - 1);
+                    projRelPath = file.getFullPath();
```

Comment:
This does not look related to the theme. Is this fixing a bug?

Reply:
Ok, I just saw the comments in the PR on this. Please separate into a different branch/PR and associate with the appropriate issue.